src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  if (typeof robot !== 'object' || robot === null || Array.isArray(robot)) {
+    throw new Error('Input must be an object');
+  }
+
+  const inverted = {};
+
+  for (const key in robot) {
+    if (inverted[robot[key]]) {
+      return null;
```

Comment:
do you really need to return null here? I believe `break` or `continue` would work better here, depending on your need

Reply:
@naziamoff [If any of the object values are repeated, return 'null'.](https://github.com/mate-academy/js_inverse_robot/blob/master/readme.md#:~:text=If%20any%20of%20the%20object%20values%20are%20repeated%2C%20return%20%27null%27.)